distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -582,3 +630,9 @@ Decimal.from (that : Text) = Decimal.parse that
 
 ## PRIVATE
 Decimal.from (that : Number) = Decimal.new that
+
+## PRIVATE
```

Comment:
Why are these conversions marked as `## PRIVATE`? I believe suggestion database doesn't contain `from` conversion methods, so the IDE shouldn't display these methods in the component browser anyway.

Reply:
Removed.